src/main.ts

```diff
@@ -185,9 +185,21 @@ async function run_linux(): Promise<void> {
   }
 }
 
+async function run_macos(): Promise<void> {
+  try {
+    core.startGroup('Installing tarantool')
+    await exec.exec(`brew install tarantool`)
```

Comment:
1. If I asked one version (say, 1.10) I may silently get another one?
2. How about caching?

Without caching and version choosing there is no much sense in the action: I can just use `brew install tarantool` directly.

Don't know, whether it'll actual for you, but if you'll build tarantool from sources on Mac OS 11, you should workaround https://github.com/tarantool/tarantool/issues/6076: remove src/lib/small/VERSION and disable generation of VERSION file in tarantool (see CMakeLists.txt:251).